src/lib/dhcpsrv/tests/cql_lease_mgr_unittest.cc

```diff
@@ -679,6 +679,14 @@ TEST_F(CqlLeaseMgrTest, lease6LeaseTypeCheck) {
     testLease6LeaseTypeCheck();
 }
 
+/// @brief Verifies the getLeases6(DUID) method
+///
+/// Adds 3 lease and verifies fetch by DUID.
+/// Verifies retrival of non existant DUID fails
+TEST_F(CqlLeaseMgrTest, getLeases6Duid) {
+   testGetLeases6Duid(); 
```

Comment:
Crash issue wasmerging old code where user_context was absent. Fixed the issue by adding user_context to the query.

Reply:
Issue was because of merging old code which did not contain user_context in the query.  Fix was to add user_context in the new query. 

Note: Google Test filter = CqlLeaseMgrTest.getLeases6Duid
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from CqlLeaseMgrTest
[ RUN      ] CqlLeaseMgrTest.getLeases6Duid
/home/mayya/projects/kea_fork/kea/src/share/database/scripts/cql/dhcpdb_drop.cql:25:OperationTimedOut: errors={'127.0.0.1': 'Client request timeout. See Session.execute[_async](timeout)'}, last_host=127.0.0.1
runCqlSchema failed:cqlsh -u keatest -p keatest -k keatest -f/home/mayya/projects/kea_fork/kea/src/share/database/scripts/cql/dhcpdb_drop.cql
/home/mayya/projects/kea_fork/kea/src/share/database/scripts/cql/dhcpdb_drop.cql:19:OperationTimedOut: errors={'127.0.0.1': 'Client request timeout. See Session.execute[_async](timeout)'}, last_host=127.0.0.1
/home/mayya/projects/kea_fork/kea/src/share/database/scripts/cql/dhcpdb_drop.cql:20:OperationTimedOut: errors={'127.0.0.1': 'Client request timeout. See Session.execute[_async](timeout)'}, last_host=127.0.0.1
/home/mayya/projects/kea_fork/kea/src/share/database/scripts/cql/dhcpdb_drop.cql:25:OperationTimedOut: errors={'127.0.0.1': 'Client request timeout. See Session.execute[_async](timeout)'}, last_host=127.0.0.1
runCqlSchema failed:cqlsh -u keatest -p keatest -k keatest -f/home/mayya/projects/kea_fork/kea/src/share/database/scripts/cql/dhcpdb_drop.cql
[       OK ] CqlLeaseMgrTest.getLeases6Duid (162208 ms)
[----------] 1 test from CqlLeaseMgrTest (162208 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (162208 ms total)
[  PASSED  ] 1 test.
mayya@mayya-fujitsu:~/